inference-engine/src/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -119,6 +120,7 @@ bool ngraph::pass::CommonOptimizations::run_on_function(std::shared_ptr<ngraph::
     decomp->add_matcher<ngraph::pass::ConvertSpaceToDepth>();
     decomp->add_matcher<ngraph::pass::BatchNormDecomposition>();
     decomp->add_matcher<ngraph::pass::MVN6Decomposition>();
+    decomp->add_matcher<ngraph::pass::SimplifyCTCGreedyDecoderSeqLen>();
```

Comment:
Why is it called in common optimizations? It should be called on plugin side.

Reply:
We discussed it with @GlebKazantaev. The transformation can be used any plugin.

Reply:
I see. Thanks!

Reply:
No.
This pass is needed by all plugins except CPU which is disable this pass.
When most of plugins will support `CTCGreedyDecoderSeqLen` then we can move this pass directly to remaining plugins which still do not support it. But now only CPU supports it, so the CommonTransformations is the right place.